### PR TITLE
Rewrite allocation charts

### PR DESCRIPTION
**Pull request by maciaszczykm**
Zrzut ekranu 2019-11-28 o 14 57 48

### CODE REVIEW

**Comment by hwdef** [comment]
/lgtm
Thanks for your contribution.

**Comment by floreks** [comment]
It would be nice to use core and memory filters that we have to show bytes/cores values with proper suffixes. billions of bytes is not very user friendly.

**Comment by maciaszczykm** [comment]
@floreks It is not possible to apply them there, unfortunately. I can change them to cores and megabytes though.

File: src/app/frontend/resource/cluster/node/detail/template.html [hunk 1]
```diff
@@ -129,91 +129,39 @@
...
-          </div>
-          <div class="kd-graph-legend-entry">
-            <div class="kd-graph-legend-placeholder"></div>
-            <span i18n>Capacity</span>
-            <span class="kd-graph-legend-value">
-              {{node?.allocatedResources.cpuCapacity | kdCores}}
-            </span>
-          </div>
-        </div>
+        <div class="kd-h1">CPU</div>
```
Comment (shu-mutou) [inline src/app/frontend/resource/cluster/node/detail/template.html]: Could you add `i18n` attribute for some elements that should be translated?

**Pull request merged**